dao/src/main/java/com/epam/esm/dao/entity/Certificate.java

```diff
@@ -0,0 +1,150 @@
+package com.epam.esm.dao.entity;
+
+import javax.persistence.*;
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+import java.util.HashSet;
+import java.util.Set;
+
+@Entity
+@Table(name = "certificates")
+public class Certificate {
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  private Long id;
+
+  @Column(name = "name")
```

Comment:
`Column` annotations is unnecessary if the field name is equal to the column name